counter/spago.dhall

```diff
@@ -0,0 +1,9 @@
+{ name =
+    "purescript-elmish-examples-counter: Counterexample"
+, dependencies =
+    [ "elmish"
```

Comment:
I'm not including any of the usual suspects here (e.g. "aff", "console", etc.), because they will come with `elmish`.

Reply:
That’s ok, but long-term I think it’s cleaner to explicitly list dependencies the examples are directly using.